doc/source/gallery/plotting_with_folium.ipynb

```diff
@@ -47,44 +54,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Create point geometries\n",
     "geometry = geopandas.points_from_xy(df.Longitude, df.Latitude)\n",
     "geo_df = geopandas.GeoDataFrame(df[['Year','Name','Country', 'Latitude', 'Longitude', 'Type']], geometry=geometry)\n",
     "\n",
     "geo_df.head()"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))\n",
-    "df.Type.unique()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "tags": [
-     "nbsphinx-thumbnail"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "fig, ax = plt.subplots(figsize=(24,18))\n",
-    "world.plot(ax=ax, alpha=0.4, color='grey')\n",
-    "geo_df.plot(column='Type', ax=ax, legend=True)\n",
-    "plt.title('Volcanoes')"
-   ]
-  },
```

Comment:
This cell contains a thumbnail used in the gallery. Can you keep it?

Reply:
Thanks for your comments. Of course, I will keep this!